packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -13,37 +15,26 @@ import {
 import type { Transaction } from '@solana/web3.js';
 import { PublicKey } from '@solana/web3.js';
 
-interface BitKeepWallet {
-    isBitKeep?: boolean;
-    connect(): Promise<void>;
-    disconnect(): Promise<void>;
-    getAccount(): Promise<string>;
-    signTransaction(transaction: Transaction): Promise<Transaction>;
-    signAllTransactions(transactions: Transaction[]): Promise<Transaction[]>;
-    signMessage(message: Uint8Array): Promise<{ signature: Uint8Array }>;
-}
-
-interface BitKeepWindow extends Window {
+interface BitgetWalletWindow extends Window {
     bitkeep?: {
-        solana?: BitKeepWallet;
+        solana?: SolanaProvider;
     };
 }
 
-declare const window: BitKeepWindow;
+declare const window: BitgetWalletWindow;
 
 export interface BitKeepWalletAdapterConfig {}
 
-export const BitKeepWalletName = 'BitKeep' as WalletName<'BitKeep'>;
+export const BitgetWalletName = WalletInfo?.name as WalletName<'Bitget Wallet'>;
```

Comment:
Please remove `Wallet` here -- this only shows up in a list of wallets, so we try to avoid repetition.

```suggestion
export const BitgetWalletName = WalletInfo?.name as WalletName<'Bitget'>;
```